support/ext/blackfire

```diff
@@ -18,11 +21,9 @@ mkdir -p ${EXT_DIR}
 mv blackfire.so ${EXT_DIR}/blackfire.so
 
 # AGENT
-blackfire_agent_version=$(curl --user-agent "Scalingo" --output agent.tar.gz \
+curl --user-agent "Scalingo" --output agent.tar.gz \
   --dump-header - --location --silent \
-  https://blackfire.io/api/v1/releases/agent/linux/amd64 \
-  | grep 'X-Blackfire-Release-Version: ' \
-  | sed "s%X-Blackfire-Release-Version: %%" | sed s%.$%%)
+  https://blackfire.io/api/v1/releases/agent/linux/amd64
```

Comment:
Don't you want to download the version `$blackfire_agent_version` here?

Reply:
It doesn't work anymore, it was a hack since the version was displayed in the header, now it's not anymore, that's why it has been frozen above.